src/watchmaker/workers/salt.py

```diff
@@ -241,7 +242,7 @@ def _get_formulas_conf(self):
         ]
 
     def _build_salt_formula(self, extract_dir):
-        if self.salt_content:
+        if self.salt_content and self.salt_content != 'None':
```

Comment:
What is the extra check here protecting us from?

Reply:
If `salt_content` is set to `None` in `config.yaml`, execution arrives at `_build_salt_formula()` with `self.salt_content` equal to the string `None`. It then tries to copy and extract a zip called `None`. This limited code changes elsewhere.

Reply:
Ahh, right, we need to fix that too. Separately, though. This kind of special handling for `'None'` is basically a user-error (ours). In yaml, it should be either an empty assignment or `null`... `salt_content:` or `salt_content: null`.